snuba/datasets/configuration/events_analytics_platform/storages/eap_items_span.yaml

```diff
@@ -1,6 +1,6 @@
 version: v1
 kind: writable_storage
-name: eap_items
+name: eap_items_span
```

Comment:
I think you'd need to change the entity as well

Reply:
I might have to change it back to `eap_items` though, we want to keep the current `eap-items-span` consumers deployed and running, and they're configured to target the `eap_items` storage.

Reply:
I think I need to push https://github.com/getsentry/snuba/pull/7124 before and switch the consumers to the renamed storage.